en/drafts/originals/space-place-gazetteers.md

```diff
@@ -25,122 +25,111 @@ doi: XX.XXXXX/phen0000
 
 ## Lesson Overview
 
-This lesson introduces you to digital gazetteers, which are spatial knowledge organization systems ((KOS) about places that record names, spatial footprints, and other characteristics historically associated with a locale. The term gazetteer refers to certain printed historical documents such as geographical indexes, directories, and encyclopdias. [^1] This lesson is focused on their digital equivalents. In brief, a gazetteer is a dictionary or list of place names. 
+The term [gazetteer](https://en.wikipedia.org/wiki/Gazetteer) refers to certain geographical documents, such as indexes, directories, and encyclopdias, that were historically printed[^1] – this lesson is focused on their digital equivalents. Digital gazetteers record names, spatial footprints, and other characteristics associated with specific places. Historical gazetteers, in particular, link discourses about one or more places over time. 
 
-A well-structured gazetteer reflects the fact that places are conceptual entities, not simply names or points on maps. Any given place may have had multiple names in numerous languages over the course of history, potentially involving conflicts about who has the power to enforce any of those names. The spatial extents, names, and feature types (settlements, buildings, nations, mountains, and so on) of places frequently change over time. 
+A gazetteer, especially a historical one, is a kind of [Knowledge Organization System (KOS)](https://en.wikipedia.org/wiki/Knowledge_organization_system), a tool 'that brings together related concepts and their names in a meaningful way, such that users of the KOS can easily comprehend the relationships represented'.[^2] The shape and organization of this KOS will be determined by the shared characteristics of the places that need to be modeled.
```

Comment:
I've moved the definition of a KOS up here, so we only need to introduce the concept once. I thought it made sense in this more general definition of a gazetteer.

Reply:
Yes. This move is okay.